clientconfig.NewServiceClient() doesn't support placement

```
opts := &clientconfig.ClientOpts{}

client, err := clientconfig.NewServiceClient("placement", opts)

if err != nil {
	panic(err)
}
```

This snippet of code leads to:
```
panic: unable to create a service client for placement
```

--- a/openstack/clientconfig/requests.go
+++ b/openstack/clientconfig/requests.go
@@ -924,6 +924,8 @@ func NewServiceClient(service string, opts *ClientOpts) (*gophercloud.ServiceCli
 		return openstack.NewObjectStorageV1(pClient, eo)
 	case "orchestration":
 		return openstack.NewOrchestrationV1(pClient, eo)
+	case "placement":
+		return openstack.NewPlacementV1(pClient, eo)
 	case "sharev2":
 		return openstack.NewSharedFileSystemV2(pClient, eo)
 	case "volume":

--- a/openstack/clientconfig/requests.go
+++ b/openstack/clientconfig/requests.go
@@ -914,6 +914,8 @@ func NewServiceClient(service string, opts *ClientOpts) (*gophercloud.ServiceCli
 		}
 	case "image":
 		return openstack.NewImageServiceV2(pClient, eo)
+	case "key-manager":
+		return openstack.NewKeyManagerV1(pClient, eo)
 	case "load-balancer":
 		return openstack.NewLoadBalancerV2(pClient, eo)
 	case "network":

